bugsnag-android-core/src/main/java/com/bugsnag/android/AppData.java

```diff
@@ -191,11 +186,8 @@ private long getMemoryUsage() {
      * Check if the device is currently running low on memory.
      */
     @Nullable
-    private Boolean isLowMemory() {
+    private Boolean isLowMemory(ActivityManager activityManager) {
```

Comment:
This doesn't need to be passed in as a parameter - it can use the member variable.

Reply:
Updated to use the field.